.github/workflows/ci.yml

```diff
@@ -0,0 +1,56 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release? yes/no'
+        default: 'no'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        run: [1, 2, 3, 4]
```

Comment:
Are we running 4 times the same thing? Is it a stability test?

Reply:
Yes. We did the same in CircleCI to detect flakiness.

Reply:
Nit: would make sense to highlight that in some kind of comment. Currently the code doesn't give any hint what is the purpose.

Reply:
Done